lib/mix/lib/mix/tasks/test.ex

```diff
@@ -210,7 +210,8 @@ defmodule Mix.Tasks.Test do
 
       mix test --only external
 
-  Which is equivalent to:
```

Comment:
What do you think about?

> For this reason, Mix also provides an `--only` option that excludes all tests and includes only the given ones:
>
>     mix test --only external
>
> Which is similar to:
>
>     mix test --include external --exclude test
>
> It differs in that the test suite will fail if no tests are executed when the `--only` option is used.

Reply:
This is much cleaner. The only small downside is that the complete explanation the behaviour of `--only`is only over after the example.